models/edr/dbt_artifacts/columns.sql

```diff
@@ -0,0 +1,38 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+
+with information_schema_columns as (
+    select
+        lower(database_name) as database_name,
+        lower(schema_name) as schema_name,
+        lower(table_name) as table_name,
+        lower(column_name) as name,
+        data_type
+    from {{ ref("information_schema_columns") }}
+),
+
+dbt_columns as (
+    select
+        lower(database_name) as database_name,
+        lower(schema_name) as schema_name,
+        lower(table_name) as table_name,
+        lower(name) as name,
+        data_type,
+        description
+    from {{ ref("dbt_columns") }}
+)
+
+select
+    database_name,
+    schema_name,
+    table_name,
+    name,
+    data_type,
+    description
+from information_schema_columns
+left join dbt_columns using (database_name, schema_name, table_name, name)
```

Comment:
I think that maybe it will be good to do `full join`, so if there are columns that only appear in `dbt_columns` they will also be included.
WDYT?

(I actually didn't know until 5 minutes ago that full join is a thing hh, wonder if it works as I think it does)

Reply:
dicussed f2f and decided we shouldn't do it (we don't want to show partial columns for tables that are not fully documented)